fhir-server/liberty-config/config/default/fhir-server-config.json

```diff
@@ -4,6 +4,7 @@
         "core": {
             "tenantIdHeaderName": "X-FHIR-TENANT-ID",
             "datastoreIdHeaderName": "X-FHIR-DSID",
+            "originalRequestUriHeaderName": "X-FHIR-FORWARDED-URL",
```

Comment:
one additional thought here... we prefix with X-FHIR, it's generally discouraged. I do like the consistency with the other headers, and wanted to just leave this comment for posterity/ future thought.

https://tools.ietf.org/html/rfc6648
https://stackoverflow.com/a/3561399/1873438

Reply:
Interesting.  I tend to agree with the first commenter on that answer:
> Just as there are many kids that will never end up as professional athletes, many custom headers will never end up as standards. I'm inclined to keep the "X-" on those.

Especially given that we already have the two others with the "X-FHIR" prefix.

Reply:
I like the consistency, but wanted to bring it up.